README.md

```diff
@@ -8,6 +8,22 @@ dependency management.
 ```sh
 pipenv sync # Installs all depedencies.
 ```
+# powershell
```

Comment:
In [Markdown](https://en.wikipedia.org/wiki/Markdown) (there are many competing Markdown specifications, but this is the one GitHub uses: https://github.github.com/gfm/), the number of hash marks determines the "level" of a heading. This matches how headings work in HTML—https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements—because Markdown is generally intended to be compatible with HTML (e.g. this README.md is automatically compiled by GitHub into the HTML you see when viewing the file on the repo homepage). So `# powershell` creates an `<h1>`-level heading, which is not really what we want; you only want one level-1 heading per page, and for Markdown files that is essentially always going to be the very first line of text.

Take a look at what this file looks like when it is rendered: https://github.com/victoria-makerspace/wa-automation/blob/56a3c8750b1e2a171dc0cfa0b18df38190fbce88/README.md
These headings are clearly out of place.

So we want these headings to be _within_ the usage section, hence we want them to be one level below. The "Usage" line has two hash marks, indicating it is a level-2 heading. So to create new sections that "belong" to the usage section, we should be using 3 hash marks. Further, the ensuing "Example" sections each "belong" to their own shell-specific subsection, so they should be level-4 headings.

E.g.:
```md
# powershell
 

 
In [Markdown](https://en.wikipedia.org/wiki/Markdown) (there are many competing Markdown specifications, but this is the one GitHub uses: https://github.github.com/gfm/), the number of hash marks determines the "level" of a heading. This matches how headings work in HTML—https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements—because Markdown is generally intended to be compatible with HTML (e.g. this README.md is automatically compiled by GitHub into the HTML you see when viewing the file on the repo homepage). So `# powershell` creates an `<h1>`-level heading, which is not really what we want; you only want one level-1 heading per page, and for Markdown files that is essentially always going to be the very first line of text.

Take a look at what this file looks like when it is rendered: https://github.com/victoria-makerspace/wa-automation/blob/56a3c8750b1e2a171dc0cfa0b18df38190fbce88/README.md
These headings are clearly out of place.

So we want these headings to be _within_ the usage section, hence we want them to be one level below. The "Usage" line has two hash marks, indicating it is a level-2 heading. So to create new sections that "belong" to the usage section, we should be using 3 hash marks. Further, the ensuing "Example" sections each "belong" to their own shell-specific subsection, so they should be level-4 headings.

E.g.:
```md
### [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/overview)
   ...

#### Example
   ...
```

Reply:
Thanks for the in depth explanation. You really are a class 1 chap Vic.